src/qibolab/platform.py

```diff
@@ -170,6 +169,283 @@ def create_tii_qw5q_gold(runcard, simulation_duration=None, address=None, cloud=
+
+    if descriptor is None:
+        descriptor = """\
+        instruments:
+            SHFQC:
+            - address: DEV12146
+              uid: device_shfqc
+            HDAWG:
+            - address: DEV8660
+              uid: device_hdawg
+            - address: DEV8673
+              uid: device_hdawg2
+            PQSC:
+            - address: DEV10055
+              uid: device_pqsc
+
+        connections:
+            device_shfqc:
+                - iq_signal: q0/drive_line
+                  ports: SGCHANNELS/0/OUTPUT
+                - iq_signal: q1/drive_line
+                  ports: SGCHANNELS/1/OUTPUT
+                - iq_signal: q2/drive_line
+                  ports: SGCHANNELS/2/OUTPUT
+                - iq_signal: q3/drive_line
+                  ports: SGCHANNELS/3/OUTPUT
+                - iq_signal: q4/drive_line
+                  ports: SGCHANNELS/4/OUTPUT
+                - iq_signal: q0/measure_line
+                  ports: [QACHANNELS/0/OUTPUT]
+                - acquire_signal: q0/acquire_line
+                  ports: [QACHANNELS/0/INPUT]
+                - iq_signal: q1/measure_line
+                  ports: [QACHANNELS/0/OUTPUT]
+                - acquire_signal: q1/acquire_line
+                  ports: [QACHANNELS/0/INPUT]
+                - iq_signal: q2/measure_line
+                  ports: [QACHANNELS/0/OUTPUT]
+                - acquire_signal: q2/acquire_line
+                  ports: [QACHANNELS/0/INPUT]
+                - iq_signal: q3/measure_line
+                  ports: [QACHANNELS/0/OUTPUT]
+                - acquire_signal: q3/acquire_line
+                  ports: [QACHANNELS/0/INPUT]
+                - iq_signal: q4/measure_line
+                  ports: [QACHANNELS/0/OUTPUT]
+                - acquire_signal: q4/acquire_line
+                  ports: [QACHANNELS/0/INPUT]
+
+            device_hdawg:
+                - rf_signal: q0/flux_line
+                  ports: SIGOUTS/0
+                - rf_signal: q1/flux_line
+                  ports: SIGOUTS/1
+                - rf_signal: q2/flux_line
+                  ports: SIGOUTS/2
+                - rf_signal: q3/flux_line
+                  ports: SIGOUTS/3
+                - rf_signal: q4/flux_line
+                  ports: SIGOUTS/4
+                - rf_signal: qc0/flux_line
+                  ports: SIGOUTS/5
+                - rf_signal: qc1/flux_line
+                  ports: SIGOUTS/6
+                - rf_signal: qc3/flux_line
+                  ports: SIGOUTS/7
+
+            device_hdawg2:
+                - rf_signal: qc4/flux_line
+                  ports: SIGOUTS/0
+
+            device_pqsc:
+                - internal_clock_signal
+                - to: device_hdawg
+                  port: ZSYNCS/4
+                - to: device_hdawg
+                  port: ZSYNCS/2
+                - to: device_shfqc
+                  port: ZSYNCS/0
+        """
+
+    controller = Zurich("EL_ZURO", descriptor, use_emulation=False)
```

Comment:
What is `EL_ZURO`?

Reply:
The name for the big Zurich cluster, it's actually never used I can change to somehting more neutral